### PR TITLE
Privacy settings

### PR DESCRIPTION
**Pull request by antaranyan**

### CODE REVIEW

**Comment by bonop320** [comment]
Add alias to include local configuration file 

Or better a script to not throw if local file does not exist

**Comment by antaranyan** [comment]
Then we lose zsh autocompletion features for `gpg` if include 2 `--options file` arguments

Enabling it manuallyis too much work for something custom.

File: gnupg/.gnupg/gpg.conf [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+# URL, then disable use of that preferred keyserver to refresh the key from
+# keyserver-options no-honor-keyserver-url
+
+# When searching for a key with --search-keys, include keys that are marked on
+# the keyserver as revoked
+# keyserver-options include-revoked
+
+
+#-----------------------------
+# photos
```
Comment (bonop320) [inline gnupg/.gnupg/gpg.conf:81]: good stuff here, son

File: git/.gitconfig [hunk 2]
```diff
@@ -45,5 +45,9 @@
 
 [credential]
   helper = cache --timeout=28800
+
 [difftool]
-	prompt = false
+  prompt = false
+
+[commit]
+  gpgSign = true
```
Comment (bonop320) [inline git/.gitconfig]: not really necessary

**Review by bonop320 — CHANGES_REQUESTED** [review]

File: git/.gitconfig (hunk 2 above)
Comment (antaranyan) [reply git/.gitconfig]: ok

**Pull request merged**